docs/maintainers/growing-your-project.md

```diff
@@ -0,0 +1,76 @@
+
+## Recruiting Team Members and Contributors Outside of Your Project
+
+If you've looked at your project's[Repository Insight Page](../features/repo-insights.md) and realize there is no one that could help you, consider recruiting new contributors. To do that, create another repository insight page with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  
+
+:::tip
+
+Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.
+
+:::
+
+Once you've identified the people you want to ask, reach out to them, providing a clear message and a list of expectations.
+
+After the people from your list has agreed to join your project, create [your team on GitHub](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team) and [sync it](../welcome/faqs.md#sync-your-github-team) to your project's Repository Insight page.
+
+![team sync gif](../../static/gif/team-sync.gif)
+
+## Understanding team participation
+
+To ensure that your open source project reaches it growth strategy goal(s), it is important that your team members actively participate in the project as much as possible. The [Contributors tab](../maintainers/maintainers-guide.md#contributors) in your project's Repository Insights page can be helpful in fostering participation. Here are some strategies you can use this tab to monitor your team member's progress:
+
+- Identify the amount of repositories they contributed to in the Repositories section to see if they are immersed in the project.
+- Take note of the date of their recent contribution in the Last Contributed section to see if it aligns with the number in the Repositories section.
+- Check the level in the Act tab to determine how often they contribute to the project
+
+:::tip
+
+If you see "Low" in the Act tab, consider messaging your team member about this information and request a meeting to discuss possible strategies to help them progress.
+
+:::
+
+## Creating Transparency and Communication With Your Team
+
+The methods  utilized to collaborate and communicate with your team play a crucial role in the growth of your open source project.  Creating and sharing your project's [workspace](../features/workspaces.md) page with your team members can be helpful in fostering a collaborative environment. After creating this page, here are some ways to leverage its information to engage with your team:
+
+- Analyze the closed number Pull Requests and Issues tabs and the PR velocity to think of new ways to quickly review and merge people's contributions.
+- Use the amount of stars and forks displayed in the Engagement tab to discuss the effectiveness of current promotion strategies and to determine new methods to promote the project.
```

Comment:
There is no Engagement "tab".

Reply:
Yes there is @adiati98 

![screenshot showing Engagement tab](https://github.com/open-sauced/docs/assets/105683440/e57855ed-f780-435a-9050-cbfe6311a27b)

Reply:
I know there's an "Engagement", but I'm not sure if it calls a "tab".

Reply:
Ah so that's the name! I think statistic card would be a good name.

Reply:
Yep. Also, it would be great if you try to search around first, then ask for clarity with some explanation of what you did and you've get so far when you're not sure about something rather than assuming. 🙂